src/main/java/com/twilio/oai/TwilioJavaGenerator.java

```diff
@@ -277,6 +290,52 @@ private void addOperationName(final CodegenOperation operation, final String nam
         operation.vendorExtensions.put("x-name", name);
         operation.vendorExtensions.put("x-name-lower", name.toLowerCase());
     }
+    private String getMd5(String input){
+        try {
+            MessageDigest md = MessageDigest.getInstance("MD5");
+            byte[] messageDigest = md.digest(input.getBytes());
+            BigInteger no = new BigInteger(1, messageDigest);
+            String hashtext = no.toString(16);
+            while (hashtext.length() < 32) {
+                hashtext = "0" + hashtext;
+            }
+            return hashtext;
+        }
+
+        // For specifying wrong message digest algorithms
+        catch (NoSuchAlgorithmException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private long calculate_serial_version_uid(final List<CodegenProperty> modelProperties){
+
+        String signature = calculate_signature(modelProperties);
+        String md5Hash  = getMd5(signature);
+        return Long.parseLong(getMd5(md5Hash).substring(0,12), 16);
+    }
+
+    private String calculate_signature(final List<CodegenProperty> modelProperties){
```

Comment:
nit: Method names should be `lowerCamelCase`.

Reply:
Fixed this